pages/desktop/search.py

```diff
@@ -36,6 +36,11 @@ def wait_for_contextcard_update(self, value):
         except NoSuchElementException:
             print('Search context card header was not loaded')
 
+    def search_results_list_loaded(self, count):
+        """method used when we need to check that the search results list
+        contains a certain number of items"""
+        self.wait.until(lambda _: len(self.result_list.extensions) > count)
```

Comment:
Can you add a message here incase this fails?

Reply:
I've added the message you suggested and modified a couple of other methods as well. 
The build is still failing but from what I can tell stage is having some general problems with response times.

Reply:
@jrbenny35 the build is passing now. If you want to have a final look or have any additional suggestions, please let me know.